### PR TITLE
fix(DPNS): error handling & type locator

### PR DESCRIPTION
**Pull request by Alex-Werner**
### Issue being fixed or implemented  

If we were to manage multiples apps with an updated dpns (with another order than [0]), creating a new names would have failed to find `domain` and `preorder` from the schema.

### What was done  

- Fix the issue by passing it the expected locator (`dpns.`)
- Impr: error handling on fetching contracts & documents

### How Has This Been Tested?

Using current test suite. 

### Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [N/A] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/DashJS/SDK/Platform/methods/documents/create.ts [hunk 1]
```diff
@@ -4,19 +4,18 @@ declare interface createOpts {
     [name:string]: any;
 }
 
-export function create(this: Platform, typeLocator: string, identity: any, opts: createOpts): any {
+export async function create(this: Platform, typeLocator: string, identity: any, opts: createOpts): Promise<any> {
     const {  dpp } = this;
```
Comment (thephez) [inline src/DashJS/SDK/Platform/methods/documents/create.ts]:
```suggestion
    const { dpp } = this;
```

File: src/DashJS/SDK/Platform/methods/documents/get.ts [hunk 2]
```diff
@@ -29,11 +29,16 @@ export async function get(this: Platform, typeLocator: string, opts: fetchOpts):
         const documents: any[] = [];
 
         for (const rawData of rawDataList) {
-            documents.push(await this.dpp.document.createFromSerialized(rawData, {skipValidation: true}));
+            try{
```
Comment (thephez) [inline src/DashJS/SDK/Platform/methods/documents/get.ts]:
```suggestion
            try {
```

File: src/DashJS/SDK/Platform/methods/documents/get.ts [hunk 3]
```diff
@@ -29,11 +29,16 @@ export async function get(this: Platform, typeLocator: string, opts: fetchOpts):
...
+                const doc = await this.dpp.document.createFromSerialized(rawData, {skipValidation: true})
+                documents.push(doc);
+            }catch (e) {
```
Comment (thephez) [inline src/DashJS/SDK/Platform/methods/documents/get.ts]:
```suggestion
            } catch (e) {
```

File: src/DashJS/SDK/Platform/methods/documents/get.ts [hunk 4]
```diff
@@ -29,11 +29,16 @@ export async function get(this: Platform, typeLocator: string, opts: fetchOpts):
...
+                console.error('Document creation: failure', e);
+            }
         }
         return documents
     }catch (e) {
```
Comment (thephez) [inline src/DashJS/SDK/Platform/methods/documents/get.ts]:
```suggestion
    } catch (e) {
```

**Comment by Alex-Werner** [comment]
It does, small style changes rarely end up in separate PR, we would have way too much of them in a early project repo :)
Thanks for them :D

**Pull request merged**